frontend/coprs_frontend/config/copr.conf

```diff
@@ -171,7 +171,8 @@ HIDE_IMPORT_LOG_AFTER_DAYS = 14
 # OIDC_TOKEN_URL=""
 # OIDC_USERINFO_URL=""
 
-
+# Whether to show a total packages count at homepage
+# PACKAGES_COUNT = False
```

Comment:
We should have this set by default (FALSE) in `config.py`

Reply:
> We should have this set by default (FALSE) in `config.py`
Done